tests/Error/MindeeHttpExceptionTest.php

```diff
@@ -95,7 +95,7 @@ public function testHTTP500Exception()
         $this->assertEquals(500, $error500->statusCode);
         $this->assertEquals("failure", $error500->apiCode);
         $this->assertEquals("Inference failed", $error500->apiMessage);
-        $this->assertEquals("Cannot run prediction: ", $error500->apiDetails);
+        $this->assertEquals("Can not run prediction: ", $error500->apiDetails);
```

Comment:
```suggestion
        $this->assertEquals("Cannot run prediction: ", $error500->apiDetails);
```

https://www.merriam-webster.com/grammar/cannot-vs-can-not-is-there-a-difference